src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/requestposthearing/RequestPostHearingSubmittedHandler.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.requestposthearing;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.requireNonNull;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.CcdCallbackMap;
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.PostHearing;
+import uk.gov.hmcts.reform.sscs.ccd.domain.RequestPostHearingTypes;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+import uk.gov.hmcts.reform.sscs.ccd.service.CcdCallbackMapService;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class RequestPostHearingSubmittedHandler implements PreSubmitCallbackHandler<SscsCaseData> {
+    private final CcdCallbackMapService ccdCallbackMapService;
+    @Value("${feature.postHearings.enabled}")
+    private final boolean isPostHearingsEnabled;
+
+    @Override
+    public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
```

Comment:
do these need to be called in the handle method? (several of these in the new files)